apps/challenges/aws_utils.py

```diff
@@ -16,6 +16,16 @@
     construct_and_send_worker_start_mail,
     construct_and_send_eks_cluster_creation_mail,
 )
+from .task_definations import (
```

Comment:
It should be `task_definitions` not `task_definations`. Please replace `definations` with `definition`

Reply:
Updated.